sys/hoon.hoon

```diff
@@ -9765,10 +9765,13 @@
     ^-  type
     ?-  -.lap
       %&  p.lap
-      %|  %-  fire
+      %|  %-  fork
           %+  turn  ~(tap in q.lap)
-          |=  {a/type b/foot}
-          [a [%dry %$ 1]]
+          |=  [a=type *]
+          :-  %hold
```

Comment:
Is there any reason this has to be a `%hold`. Is there any distinction between expanding `[%hold [%core ...] %$ 1]` and just producing the `%core` itself?

Reply:
No, I don't think so.  I only left it in out of conservatism (e.g., matching the pattern in +fire), but after testing it a bit there doesn't seem to be any reason to lose it.